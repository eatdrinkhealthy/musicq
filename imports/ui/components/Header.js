// @flow
import React from "react";

// <img src="/images/music_note.png" alt="music note" />
const Header = (): React.Element<*> => (
  <div>
    <h1>MusicQ</h1>
    <div className="row">
      <div className="six columns">Side A</div>
      <div className="six columns">Side B</div>
    </div>
  </div>
);

export default Header;
