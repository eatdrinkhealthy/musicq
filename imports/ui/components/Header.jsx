// @flow
import React from "react";
import { Link } from "react-router";

// <img src="/images/music_note.png" alt="music note" />
const Header = (): React$Element<*> => (
  <div className="row header">
    <div className="six columns">
      <h1>MusicQ</h1>
    </div>
    <div className="six columns">
      <Link to="/" className="u-pull-right">
        <div className="burger">
          <div className="burger-bar" />
          <div className="burger-bar" />
          <div className="burger-bar" />
        </div>
      </Link>
    </div>
  </div>
);

export default Header;
