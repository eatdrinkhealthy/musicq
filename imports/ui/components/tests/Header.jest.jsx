// @flow
/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */
import React from "react";
import renderer from "react-test-renderer";
import Header from "../Header";

describe("<Header />", function () {
  it("matches render snapshot", function () {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
