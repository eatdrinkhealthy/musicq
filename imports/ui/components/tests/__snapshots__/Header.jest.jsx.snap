exports[`<Header /> matches render snapshot 1`] = `
<div
  className="row header">
  <div
    className="six columns">
    <h1>
      MusicQ
    </h1>
  </div>
  <div
    className="six columns">
    <a
      className="u-pull-right"
      onClick={[Function]}
      style={Object {}}>
      <div
        className="burger">
        <div
          className="burger-bar" />
        <div
          className="burger-bar" />
        <div
          className="burger-bar" />
      </div>
    </a>
  </div>
</div>
`;
